packages/api/src/schema/schema.yaml

```diff
@@ -1070,6 +1090,8 @@ components:
           example: filename.mp4
         profile:
           $ref: "#/components/schemas/ffmpeg-profile"
+        clip:
```

Comment:
To evaluate if ffmpeg-trim and ffmpeg-profile can belong to the same object, and if it's possible to create a clip in keyframes without re-encode the asset (currently ffmpeg-profile is required, could be optional)

Reply:
I think either here or in `task-runner`, we could default the `profile` to something like "source" (`-c copy`) in case its not present 🤔